build/root/WORKSPACE

```diff
@@ -106,6 +67,14 @@ container_pull(
     tag = "latest",  # ignored, but kept here for documentation
 )
 
+load("//build:workspace.bzl", "cni_tarballs", "cri_tarballs", "debian_image_dependencies")
```

Comment:
Can we provide a method that does all three of these things?

Reply:
done